src/screens/UserPortal/Settings/Settings.tsx

```
@@ -419,6 +353,24 @@ export default function settings(): JSX.Element {
                     </Col>
+                    <Col lg={4}>
+                      <Form.Label
+                        htmlFor="inputPassword"
+                        className={`${styles.cardLabel}`}
+                      >
+                        {tCommon('password')}
+                      </Form.Label>
+                      <Form.Control
+                        type="password"
+                        id="inputPassword"
+                        placeholder='enter new password'
```

ESLint

[error] Replace `'enter·new·password'` with `"enter·new·password"`

(prettier/prettier)